test/setup/recording.ts

```diff
@@ -0,0 +1,52 @@
+// When this integration switches to the new SDK, simply change the import source for the following imports
+import {
+  setupRecording,
+  Recording,
+  SetupRecordingInput,
+  RecordingEntry,
+} from '@jupiterone/integration-sdk-testing';
+import { gunzipSync } from 'zlib';
+
+export { Recording };
+
+export function setupOktaRecording(
+  input: Omit<SetupRecordingInput, 'mutateEntry'>,
+): Recording {
+  return setupRecording({
+    ...input,
+    mutateEntry: mutateRecordingEntry,
+  });
+}
+
+function mutateRecordingEntry(entry: RecordingEntry): void {
```

Comment:
I think this can be replaced by https://github.com/JupiterOne/sdk/blob/ee9ec9ae9df9b9d6fd707628a24b1a97a4a15467/packages/integration-sdk-testing/src/recording.ts#L141